app/src/contacts/OrganizationController.js

```diff
@@ -2,36 +2,65 @@
 angular
         .module('openeApp.contacts')
         .controller('OrganizationController', OrganizationController);
-
-function OrganizationController($filter, $stateParams, $state, $mdDialog, $location, $translate, $timeout, VirtualRepeatLoader,
-        contactsService, notificationUtilsService, organizationDialogService) {
+function OrganizationController($scope, $stateParams, $state, $mdDialog, $translate,
+        contactsService, notificationUtilsService, organizationDialogService, $controller, filesService) {
     var vm = this;
     vm.parentState = $state.current.name.split('.')[0];
     vm.showOrganizationEdit = showOrganizationEdit;
     vm.deleteOrganization = deleteOrganization;
-    vm.filterArray = {};
-    vm.columnFilter = columnFilter;
-    vm.organizations = [];
-    vm.loadOrganizations = loadOrganizations;
-
-    if ($stateParams.uuid) {
-        //infoForm
-        initInfo();
-    } else {
-        //list
-        vm.loadOrganizations();
+    initInfo();
+    initFiles();
+    function initInfo() {
+        contactsService.getOrganization($stateParams.storeProtocol, $stateParams.storeIdentifier, $stateParams.uuid).then(function(organization) {
+            vm.organization = organization;
+        });
     }
 
-    function loadOrganizations(){
-        contactsService.getOrganizations(vm.searchQuery).then(function(result){
-            vm.organizations = result.items;
+    function initFiles() {
+        $scope.filesVm = {};
```

Comment:
Seems like would be better to extract this controller into separate OrganizationFilesController.